package.json

```diff
@@ -17,8 +17,8 @@
     "test:integration": "yarn build && cd ./__integrations__ && yarn install --frozen-lockfile && yarn start",
     "test:all": "yarn test && yarn test:integration",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
-    "generate:all": "npm-run-all generate:*",
-    "generate:definitions": "gen-api-models --api-spec ./openapi/index.yaml --out-dir ./generated/definitions",
+    "generate": "npm-run-all generate:*",
+    "generate:definitions": "rimraf ./generated && gen-api-models --api-spec ./openapi/index.yaml --out-dir ./generated/definitions",
     "generate:templates": "gulp generate:templates",
```

Comment:
I've an issue running this command locally
```
$ gulp generate:templates
fs.js:45
} = primordials;
    ^

ReferenceError: primordials is not defined
```
Seems that the current version of gulp is incompatible with the nodejs runtime 12

Reply:
Adding this resolution into the `package.json` file should solve the problem
```
"resolutions": {
    "graceful-fs": "^4.2.4"
  }
```